Mathlib/LinearAlgebra/Multilinear/Curry.lean

```diff
@@ -316,7 +343,7 @@ def curryFinFinset {k l n : ℕ} {s : Finset (Fin n)} (hk : #s = k) (hl : #sᶜ
     MultilinearMap R (fun _ : Fin n => M') M₂ ≃ₗ[R]
       MultilinearMap R (fun _ : Fin k => M') (MultilinearMap R (fun _ : Fin l => M') M₂) :=
   (domDomCongrLinearEquiv R R M' M₂ (finSumEquivOfFinset hk hl).symm).trans
-    (currySumEquiv R (Fin k) M₂ M' (Fin l))
```

Comment:
I think explicit arguments here were probably the right call, though the order here was silly.

Reply:
At the very least I think `R` should be explicit here, since trying to write large compositions can be quite painful if Lean cannot guess the ring.

Mainly though my argument would be that:
* `domDomCongrLinearEquiv` and `multilinearCurryRightEquiv` and a bunch of other equivs in the file take many explicit arguments
* Removing the arguments is a (mildly) breaking change for downstream code, and probably makes more sense in a PR that is explicitly about such a change, for ease of discovery in the blame.